packages/ember/addon/instance-initializers/sentry-performance.ts

```diff
@@ -37,7 +37,7 @@ function getBackburner() {
 
 function getTransitionInformation(transition: any, router: any) {
   const fromRoute = transition?.from?.name;
-  const toRoute = transition && transition.to ? transition.to.name : router.currentRouteName;
+  const toRoute = transition && transition.to ? transition.to.name.toString() : router.currentRouteName;
```

Comment:
Let’s avoid changing ember for now since the router should always return a string? We can come back to this later on.

Reply:
Fixed